src/licenses/index.ts

```diff
@@ -121,6 +122,28 @@ export default class Licenses extends Base {
     return this._request<ICheckLicensesCapabilities>(`${RESOURCE_NAMES.LICENSES}/check?${params}`);
   }
 
+  /**
+   *  Checks a license's capabilities (Version 2)
+   *
+   * @param {string} productUuid - The UUID of the product to check the license against
+   * @param {string[]} granteeIds - The grantee IDs to check the license for
+   * @param {number} grace - The number of days to extend the end dates of capabilities
+   *
+   * @returns {Promise<ICheckLicensesCapabilitiesV2>} The capabilities of the license passed
+   */
+  public checkV2(
```

Comment:
Looking at the requests it looks like the only difference is the `v2` header we pass in. I wonder if it's worth us adding a `options` object to the original `check` method that allows people to opt into `v2` to save duplicating the method?

Alternatively, if we want to keep them separate could we come up with a different name that doesn't involve putting the version number in the name. Feels like it's a slippery road to having several called `v2`, `v3`, `v4`, etc and then we're effectively stuck with those names as remaining them back to `check` is a breaking change needing a new major release.

Also if we are keeping them separate we should mark the first one as deprecated and say it'll be removed in the future and push people to use the new one in the deprecation message.